mod3hw/bonusHW/index.html

```diff
@@ -5,40 +5,63 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Module 3 Bonus HomeWork</title>
-    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/style_flexbox.css">
 </head>
 <body>
-    <div class="left-logo_palm">
-        <img class="" src="img/palm tree.png" alt="">
-    </div>
-    <div class="left-logo_sloghan">
-        <img class="" src="img/logo.png" alt="">
-    </div>
-    <p class="asia-travel">Путешествия по Азии</p>
-    <div class="contacts">
-        <ul class="contacts_list">
-            <li>Contact Us</li>
-            <li>+12345678910</li>
-            <li>info@asia-holidays.com</li>
-        </ul>
-    </div>
-    <div class="first-row-pictures">
-        <ul class="first-row-pictures_list">
-            <li><img src="img/india_photo.png" alt=""></li>
-            <li><img src="img/nepal_photo.png" alt=""></li>
-            <li><img src="img/tibet_photo.png" alt=""></li>
-        </ul>
-    </div>
-
-
-    <div class="second-row-pictures">
-        <ul class="second-row-pictures_list">
-            <li><img src="img/bhutan_photo.png" alt=""></li>
-            <li><img src="img/sri-lanka photo.png" alt=""></li>
-            <li><img src="img/maldives_photo.png" alt=""></li>
-        </ul>
-    </div>
-
-
+    <main>
+        <div class="logo-contacts">
+            <div class="left-logo">
+                <img class="left-logo_palm" src="img/palm tree.png" alt="palm-logo">
+                <img class="left-logo_sloghan" src="img/logo.png" alt="sloghan-logo">
+                <p class="asia-travel">Путешествия по Азии</p> 
+            </div>      
+            <ul class="contacts_list">
+                <li>Contact Us</li>
+                <li><a href="tel:+12345678910">+12345678910</a></li>
+                <li><a href="mailto:info@asia-holidays.com">info@asia-holidays.com</a></li>
+            </ul>
+        </div>
+        <div class="main-content__pictures">
```

Comment:
`main` will be here if we use `header` for logo and contacts. don't forget to rename class